main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
why two main.go?

Reply:
it was too test something, mb forgot to delete